pyamg/aggregation/smooth.py

```diff
@@ -18,7 +18,7 @@
 
 
 # Satisfy_Constraints is a helper function for prolongation smoothing routines
-def Satisfy_Constraints(U, B, BtBinv, cost=[0]):
+def Satisfy_Constraints(U, B, BtBinv, cost=[0.0]):
```

Comment:
Note that none of the 0.0's are necessary, because if cost isn't passed in, then it's not passed out either.